static/app/views/settings/organizationDeveloperSettings/resourceSubscriptions.tsx

```diff
@@ -49,6 +42,10 @@ export default class Subscriptions extends Component<Props> {
     if (JSON.stringify(events) !== JSON.stringify(permittedEvents)) {
       this.save(permittedEvents);
     }
+
+    if (this.props.webhookDisabled && this.props.events.length) {
+      this.save([]);
```

Comment:
what happens if both this save and the save above are called when `events` has changed and `webhookDisabled` is true? maybe check multiple `save` are safe.